rotkehlchen/constants.py

```diff
@@ -5,6 +5,7 @@
 
 ETH_DAO_FORK_TS = 1469020840  # 2016-07-20 13:20:40 UTC
 BTC_BCH_FORK_TS = 1501593374  # 2017-08-01 13:16:14 UTC
+XRB_NANO_REBRAND_TS = 1525132800.0  # 2018-05-01 2:00:00 UTC
```

Comment:
Rotkehlchen uses unix timestamps (integers). Can you remove the decimal? Also ... is the time AM/PM? If possible write it in the 24:00 format.

Reply:
Done